README.md

```diff
@@ -46,33 +38,37 @@
 - [WWDC](https://github.com/insidegui/WWDC) - The WWDC app. ![Open-Source Software][OSS Icon]
 - [Zsh](http://www.zsh.org/) -  A powerful command line shell. ![Open-Source Software][OSS Icon]
 
+### Design
+- [Sketch](http://bohemiancoding.com/sketch/) - An awesome vector editor like Adobe Illustrator. Sketch gives you the power, flexibility and speed you always wanted in a lightweight and easy-to-use package. Finally you can focus on what you do best: Design.
+- [PaintCode](http://www.paintcodeapp.com) - A powerful tool to turn your vector drawings into code.
 
 ### Editors
-
 - [Atom](https://atom.io/) - The hackable text editor. ![Open-Source Software][OSS Icon]
 - [Brackets](http://brackets.io/) - A modern text editor that understands web design. ![Open-Source Software][OSS Icon]
+- [CotEditor](http://coteditor.com) - Lightweight Plain-Text Editor for OS X. ![Open-Source Software][OSS Icon]
 - [Light Table](https://github.com/LightTable/LightTable) - A customizable editor with REPL support for Clojure, JavaScript, Python and others. ![Open-Source Software][OSS Icon]
 - [Macvim](https://github.com/b4winckler/macvim) - Vim, the text editor. ![Open-Source Software][OSS Icon]
+- [Mou](http://25.io/mou/) A Lightweight Markdown editor.
 - [Sublime Text 3](http://www.sublimetext.com/) - The sophisticated text editor.
 - [Textmate](https://macromates.com/) - A graphical text editor. ![Open-Source Software][OSS Icon]
 - [Visual Studio Code](https://code.visualstudio.com/) - Build and debug modern web and cloud applications.
 
-
 ### Finder
-
 - [Path Finder](http://www.cocoatech.com/pathfinder/) - A powerful dual-pane browser alternative to Finder.
 - [TotalFinder](http://totalfinder.binaryage.com/) - A powerful alternative to Finder.
 - [XtraFinder](https://www.trankynam.com/xtrafinder/) - Adds useful features to Finder.
 
-
 ### Games
 - [OpenEmu](http://openemu.org/) - Multiple Video Game System. ![Open-Source Software][OSS Icon]
 - [Screentendo](https://github.com/AaronRandall/Screentendo) - Turn your screen into a playable level of Mario. ![Open-Source Software][OSS Icon]
 - [stockfish-mac](https://github.com/daylen/stockfish-mac) - Beautiful, powerful chess application. ![Open-Source Software][OSS Icon]
 
+### Movies
+- [Popcorn Time](https://popcorntime.io) - An app to watch Movies and TV Shows instantly. ![Open-Source Software][OSS Icon]
+- [Transmission](http://www.transmissionbt.com) - An open source, volunteer-based project. Unlike some BitTorrent clients, Transmission doesn't play games with its users to make money. ![Open-Source Software][OSS Icon]
+- [VLC](http://www.videolan.org/vlc/index.html) - Simple, fast and powerful media player that has a native UI. ![Open-Source Software][OSS Icon]
```

Comment:
I don't think VLC fits into this list, though it is an awesome product but still it is a very trivial product and very well known.